pkg/server/imagefs_info.go

```diff
@@ -26,5 +26,26 @@ import (
 
 // ImageFsInfo returns information of the filesystem that is used to store images.
 func (c *criContainerdService) ImageFsInfo(ctx context.Context, r *runtime.ImageFsInfoRequest) (*runtime.ImageFsInfoResponse, error) {
-	return nil, errors.New("not implemented")
+	snapshots := c.snapshotStore.List()
+	timestamp := time.Now().UnixNano()
+	var usedBytes, inodesUsed uint64
+	for _, sn := range snapshots {
+		// Use the oldest timestamp as the timestamp of imagefs info.
+		if sn.Timestamp < timestamp {
+			timestamp = sn.Timestamp
+		}
+		usedBytes += sn.Size
```

Comment:
was the response supposed to be an aggregate response? If so why did they create an array of filesystemusage? Or is that what the below todo is for?

Reply:
Because there may be multiple filesystems using for image management.